tensorflow/python/keras/backend_test.py

```diff
@@ -1736,5 +1736,40 @@ def test_batch_normalization(self):
     self.assertEqual(normed.shape.as_list(), [10, 3, 5, 5])
 
 
+class ControlOpsTests(test.TestCase):
+  @test_util.run_in_graph_and_eager_modes
+  def test_function_switch_basics(self):
+    x = ops.convert_to_tensor(np.array([[1, 2, 3], [4, 5, 6]]),
+                              dtype='float32')
+    y1 = ops.convert_to_tensor(np.array([[1, 2, 3], [4, 5, 6]]),
+                               dtype='float32')
+    x1 = keras.backend.square(x)
+    def true_fun():
+      output = x * x
+      return output
+
+    def false_fun():
+      return y1
+
+    x2 = keras.backend.switch(keras.backend.equal(x, y1), true_fun, false_fun)
```

Comment:
You could simply this code considerably by having:

condition: `array_ops.constant(True)` or `array_ops.constant(False)` (or a boolean placeholder in order to test dynamic behavior)
And for one of the functions, you can use for example: `true_fn=lambda x: array_ops.constant(True)` (easier to check the output)